### ISSUE TITLE
Podcast - FAILED TO QUERY METADATA

### ISSUE THREAD

**Issue by ajaxniix** [jsavargas/zspotify#155]
Running

`zspotify -fs https://open.spotify.com/show/5OehdZdePuUwWUS0Og9cvQ?si=483b905f21424d6f`

Results in:

```
###   get_track_info - FAILED TO QUERY METADATA   ###
track_id: 5ciiOdwOaDP2s15jF8FUgd
'NoneType' object is not subscriptable
Skipping 5ciiOdwOaDP2s15jF8FUgd - Could not get track info
###   get_track_info - FAILED TO QUERY METADATA   ###
track_id: 0vnDpe0uQgE4e3XDvEWjEa
'NoneType' object is not subscriptable
Skipping 0vnDpe0uQgE4e3XDvEWjEa - Could not get track info
###   get_track_info - FAILED TO QUERY METADATA   ###
track_id: 6h60P3VyBgVX54Fv24kTaL
'NoneType' object is not subscriptable
Skipping 6h60P3VyBgVX54Fv24kTaL - Could not get track info
###   get_track_info - FAILED TO QUERY METADATA   ###
track_id: 4Kp1gzTE9eNGslv4iwPjWH
'NoneType' object is not subscriptable
Skipping 4Kp1gzTE9eNGslv4iwPjWH - Could not get track info
###   get_track_info - FAILED TO QUERY METADATA   ###
track_id: 2Mqvug5uMGg65U8yxvIarS
'NoneType' object is not subscriptable
Skipping 2Mqvug5uMGg65U8yxvIarS - Could not get track info
###   get_track_info - FAILED TO QUERY METADATA   ###
track_id: 70uz2NmOEyViX2cMNe2Q4h
'NoneType' object is not subscriptable
Skipping 70uz2NmOEyViX2cMNe2Q4h - Could not get track info
```

**Comment by CharlieHorton** [comment]
Bump, getting exactly the same. Seems to only happen when running on my Ubuntu server, running locally on MacOS works for the same episode.

###   get_track_info - FAILED TO QUERY METADATA   ###
track_id: 0XtDijQpzDfbhi3oo46e7Z
'NoneType' object is not subscriptable
Skipping 0XtDijQpzDfbhi3oo46e7Z - Could not get track info

**Comment by jamesdeluk** [comment]
Same on Windows 11.

Trying with a single episode:
`Skipping 3urJ6r8qkTgr52jhlKGKiZ - Could not get track info`
(which makes sense, as it calls `get_track_info`)

I tried to do a little debugging myself with no success; the response seems fine.

**Comment by ResidentChief** [comment]
Same result here, windows 11, fresh python venv + ffmpeg - downloading albums works fine

**Comment by onupirat** [comment]
Hmm...no answer from the dev. Anyone figured that out yet?

**Comment by lanten111** [comment]
Yup, have sorted the issue and i'm able to download podcasts.. i didn't submit a pull request as this was my first python experience and may have messed things

Forked the repo instead, you can test it here.. https://github.com/lanten111/zspotify

**Comment by M05QU170** [comment]
> Yup, have sorted the issue and i'm able to download podcasts.. i didn't submit a pull request as this was my first python experience and may have messed things
> 
> Forked the repo instead, you can test it here.. https://github.com/lanten111/zspotify

It downloads, tags and converts but unfortunately it converts to 160kbps unnecessary, because podcasts comes with 96kpbs quality.

**Comment by obscenelyvague** [comment]
> Forked the repo instead, you can test it here.. https://github.com/lanten111/zspotify
@niclimcy Will that fix or a different fix make it on to this repo? Podcast downloading is completely broken right now.
